client/src/elements/item-preview/item-preview.js

```diff
@@ -149,8 +149,12 @@ export class ItemPreview {
         this.errorMessage = undefined;
         this.renderingInfo = renderingInfo;
       })
-      .catch(errorMessage => {
-        this.errorMessage = errorMessage;
+      .catch(response => {
+        if (response.status === 400) {
```

Comment:
Mmh. Imho it's ok to show the bare response status in this case as it should not happen. If we get a screenshot as error message from users it's clear what happened.
I thought about special handling for `Bad Request` as this is what lot's of users will see before they add the required `title` property for example.